chord_metadata_service/chord/models.py

```diff
@@ -116,6 +117,8 @@ def n_of_tables(self):
                               help_text="The grant(s) which funded and supported the work reported by the dataset.")
     keywords = ArrayField(JSONField(null=True, blank=True), blank=True, null=True,
                           help_text="Tags associated with the dataset, which will help in its discovery.")
+    version = models.CharField(max_length=200, blank=True, default=f"version_{datetime.datetime.now()}",
```

Comment:
I believe this will become a problem, the default value being interpreted once, when the model is loaded in the application. For more info : https://stackoverflow.com/questions/2771676/django-datetime-issues-default-datetime-now

I think a potential fix could be to create a lambda which will produce the string "version_..."

Reply:
The idea of default value I had in mind here:  if the version is not provided, then add version name with a datestamp when the dataset was created - creating a string 'version_{of this date}', and this value should stay persistent or the data provider can provide their own version.

Reply:
Yes that's fine except if the version is not provided, the default will stay the same until the uwsgi or gunicorn server is reloaded, as in, default will be "version_{value of datetime.now() at the moment this module is imported}".

So default should receive a method for this value to reflect the actual datetime.now() every time a default value is needed otherwise currently it's merely a string, which will stay the same : https://docs.djangoproject.com/en/3.0/ref/models/fields/#default